_tags/template.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Template
+description: An overview of template tags in the Liquid template language.
+redirect_from:
+  - /tags/comment/
+  - /tags/raw/
+---
```

Comment:
Awesome! You did what I wanted in https://github.com/Shopify/liquid/issues/1409#issuecomment-792962568 😄